pkg/packagekit/updates.jsx

```diff
@@ -23,6 +23,9 @@ var moment = require("moment");
 var Tooltip = require("cockpit-components-tooltip.jsx").Tooltip;
 require("listing.less");
 
+// for interactively testing layout changes with large updates
+//import { injectMockUpdates } from "./mock-updates.es6";
```

Comment:
Since the code fragment isn't included at all when compiled like this, I think it's a good way of including debug code. Is it entirely necessary, though? In other cases we've added debug content to the tests along with instructions on how to use it. This here is right on the verge of polluting production code with debugging data.